woocommerce/woocommerce-google-analytics-integration
# Retire `phpcs-diff`

### Changes proposed in this Pull Request:

This PR retires `phpcs-diff` and add the `phpcs` workflow.

### Checks:


### Detailed test instructions:

1. `npm run lint:php` to see if the check can pass
2. View [the check result](https://github.com/woocommerce/woocommerce-google-analytics-integration/actions/runs/9710643284/job/26801748611) of the `phpcs` workflow
3. View [the annotations of linting errors](https://github.com/woocommerce/woocommerce-google-analytics-integration/pull/446/files/0e2ce9b5ef98549a83d8d1dd581622ddfc790211) added by the `phpcs` workflow
   ![image](https://github.com/woocommerce/woocommerce-google-analytics-integration/assets/17420811/89a562fa-414a-4b64-b975-d2fd788ca68a)

### Changelog entry
